src/back/auth/src/service/session.cpp

```diff
@@ -31,7 +33,9 @@ model::Session Session::Create(
 {
 	auto session = prepare(tokens, data, userAgent, exp);
 
-	_table.Save(session);
+	auto trx = _rep.WithTrx();
+	trx.Session().Save(session);
+	trx.Commit();
```

Comment:
Транзакции нужны только там где выполняется несколько запросов отдельно. Посмотри везде пожалуйста.
```suggestion
	_rep.Session().Save(session);
```

Reply:
Проверил и поправил везде.